### DISCUSSION TITLE
Serum Proteomics and Serum Albumin

### DISCUSSION THREAD

**Discussion by DarioS** [vdemichev/DiaNN#146] [General]
I see the user guide has links to two articles in Cell Systems but I was curious if any special settings should be thought about for normalisation in this scenario. My understanding is that ALBU_HUMAN might be as much as 80% of the proteome, so the normalisation would strongly be driven by it.

**Comment by vdemichev** [comment]
Yes, in some cases it might make sense to also perform protein-level normalisation in R/Python using a package specifically meant for this purpose. In case of our work, if I recall correctly, about 180 precursors are being quantified for albumin, out of 6000 total, so the impact of specifically albumin should be minimal. But yes, agree, normalisation can be slightly biased towards several abundant proteins.

**Reply by DarioS** [reply]
I recently read [The Negative Association Between Serum Albumin Levels and Coronary Heart Disease Risk in Adults Over 45 Years Old: A Cross-sectional Survey](https://www.nature.com/articles/s41598-023-27974-w), _Scientific Reports_. My understanding is that DIA-NN employs a total sum scaling approach to make abundances comparable between samples. Would it be beneficial to offer an option to specify a list of endogenous (e.g. housekeeping proteins) or exogenous (e.g. spike-in standards) controls and only use those for rescaling of the abundances?

**Reply by vdemichev** [reply]
First that, and then more fancy normalisation. Yes, further normalisation based on a selected (large) set of proteins might make sense.

**Reply by DarioS** [reply]
Can Total Sum Scaling be turned off in G.U.I. and something like median scaling chosen instead to reduce impact of most abundant?

**Reply by vdemichev** [reply]
No, DIA-NN does sth more fancy on top of total sum scaling. But one can always perform normalisation of choice in R or Python.
